src/qt/forms/sendcoinsdialog.ui

```diff
@@ -1106,12 +1106,12 @@
             </layout>
            </item>
            <item>
-            <widget class="QCheckBox" name="optInRBF">
+            <widget class="QCheckBox" name="disableRBF">
              <property name="text">
-              <string>Allow increasing fee</string>
+              <string>Disable Replace-By-Fee</string>
              </property>
              <property name="toolTip">
-              <string>This allows you to increase the fee later if the transaction takes a long time to confirm. This will also cause the recommended fee to be lower. ("Replace-By-Fee", BIP 125)</string>
+              <string>Not using Replace-By-Fee (BIP-125) marks the transaction as final. This will also cause the recommended fee to be higher.</string>
```

Comment:
Heh, I'm less a fan of the copy here now. Maybe "Not using Replace-By-Fee (BIP-125) prevents you from increasing a transaction's fee after it is sent. This may cause the recommended fee to be higher to compensate for the increased risk a transaction is delayed." or something like that?

Reply:
I changed it to:
> Without Replace-By-Fee (BIP-125) you can't increase a transaction's fee after it is sent. A higher fee may be recommended to compensate for increased transaction delay risk.